### PR TITLE
Tutorial modal walkthrough

### PR DESCRIPTION
**Pull request by OAGr**

### CODE REVIEW

**Comment by mmcdermott** [comment]
If I download this branch, it complains about missing files and doesn't work. Are you sure all the files are properly synced (e.g. not just deleted from git or something)?

**Comment by mmcdermott** [comment]
Please remove old images, and rename the ones you are using to not have numerals, as we should only keep one copy of each.

File: src/components/spaces/show/Tutorial/index.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+import {ButtonClose} from 'gComponents/utility/buttons/close/index'
+import {TutorialMetricPage, TutorialMetricActionsPage, TutorialFunctionPage, TutorialMoreFeaturesPage} from './pages'
+import {GeneralModal} from 'gComponents/utility/modal/index'
+
+import Icon from 'react-fa'
+
+import './style.css'
+
+export const TutorialModal = ({onClose}) => {
```
Comment (mmcdermott) [inline src/components/spaces/show/Tutorial/index.js]: Not sure two components are needed given you made `GeneralModal`

File: src/components/spaces/show/Tutorial/pageStyle.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      .exampleFunction {
+        background: #eee;
+        border-radius: 0.2em;
+        padding: 0.15em 0.3em;
+        font-family: "Courier New", Courier, monospace;
+        font-size: 0.9em;
+      }
+
+      .exampleInput {
+        background-color: #c9e8c0;
```
Comment (mmcdermott) [inline src/components/spaces/show/Tutorial/pageStyle.css]: Might be good to pull this into a var and share it with the styles for the text input distribution editor decorators too.

File: src/components/spaces/show/Tutorial/pages.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
+import React from 'react'
+
+import Icon from 'react-fa'
+
+import './pageStyle.css'
+import MakeAMetric from '../../../../assets/tutorial/MakeAMetric3.gif'
```
Comment (mmcdermott) [inline src/components/spaces/show/Tutorial/pages.js]: we should make a 'gAssets' tag to simplify these.

**Comment by mmcdermott** [comment]
Why did you get rid of the links to the documentation sections? I feel like 'learn more' links would be appropriate here.

File: src/components/spaces/show/index.js [hunk 4]
```diff
@@ -358,7 +367,7 @@ export default class SpacesShow extends Component {
 
   render() {
     const space = this.props.denormalizedSpace
-    const {organizationHasFacts} = this.props
+    const {organizationHasFacts, me} = this.props
```
Comment (mmcdermott) [inline src/components/spaces/show/index.js]: Why are you doing this?

**Comment by OAGr** [comment]
I think the 'learn more' link wasn't very explanatory.  The text would need to be revised, and it would have to get repositioned, which would be a bit messy.  

I want to deploy this, or at least talk to more people about it, before changing it much.

File: src/components/utility/modal/index.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import React, {Component, PropTypes} from 'react'
+import Modal from 'react-modal'
+
+        //display: 'flex',
```
Comment (mmcdermott) [inline src/components/utility/modal/index.js]: Do you want these comments to stay for some reason?

File: src/components/utility/modal/index.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        //alignItems: 'center',
+        //justifyContent: 'center',
+        //backgroundColor: 'rgba(0,0,0,0)',
+        //border: 'none',
+        //padding: '0px',
+        //top: 0,
+        //left: 0,
+        //right: 0,
+        //bottom: 0,
+export class GeneralModal extends Component {
```
Comment (mmcdermott) [inline src/components/utility/modal/index.js]: Make functional components.

File: src/components/utility/modal/index.js [hunk 7]
```diff
@@ -0,0 +1,44 @@
+import React, {Component, PropTypes} from 'react'
```
Comment (mmcdermott) [inline src/components/utility/modal/index.js]: You don't need the PropTypes import (or Component, if you make the export functional)

**Pull request merged**